File: src/main/java/io/nayasis/sample/registrylock/redis/BootApplication.java
package io.nayasis.sample.registrylock.redis;

import lombok.extern.slf4j.Slf4j;
import org.springframework.boot.CommandLineRunner;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cache.annotation.EnableCaching;
import org.springframework.context.annotation.aspectj.EnableSpringConfigured;

@EnableCaching
@EnableSpringConfigured
@SpringBootApplication
@Slf4j
public class BootApplication implements CommandLineRunner {

    public static void main( String[] args ) {
        SpringApplication.run( BootApplication.class, args );
    }

    @Override
    public void run( String... args ) throws Exception {

    }

}

File: src/main/kotlin/com/github/nayasis/sample/distributedlock/BootApplication.kt
package com.github.nayasis.sample.distributedlock

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableScheduling

@EnableScheduling
@SpringBootApplication
class BootApplication

fun main(args: Array<String>) {
    runApplication<BootApplication>(*args)
}